.github/workflows/common.sh

```diff
@@ -2,14 +2,18 @@
 
 set -euo pipefail
 
-readonly SDK_OUTER_TOPDIR="${HOME}/flatcar-sdk"
+readonly SDK_OUTER_TOPSCRIPTSDIR="${HOME}/flatcar-sdk/scripts"
+readonly SDK_OUTER_TOPDIR="${SDK_OUTER_TOPSCRIPTSDIR}/sdk_container"
 readonly SDK_OUTER_SRCDIR="${SDK_OUTER_TOPDIR}/src"
 readonly SDK_INNER_SRCDIR="/mnt/host/source/src"
 
 readonly BUILDBOT_USERNAME="Flatcar Buildbot"
 readonly BUILDBOT_USEREMAIL="buildbot@flatcar-linux.org"
 
-function enter() ( cd ../../..; exec cork enter -- $@ )
+function enter() {
+  ${SDK_OUTER_TOPSCRIPTSDIR}/run_sdk_container -n "${PACKAGES_CONTAINER}" \
+    -C "${SDK_NAME}" $@
```

Comment:
```suggestion
    -C "${SDK_NAME}" "$@"
```
Not sure why it wasn't there before, I would preserve the parameters instead of splitting them on whitespace.